markdown/chapters/meandeck/interaction.md

```diff
@@ -522,10 +550,66 @@ cantrips. You can also build a pile around it *if* you can predict when the
 opponent will use it however that can be tricky. Also be mindful of timing for
 your opponent being able to *snipe* an LED with the artifact destruction mode.
 
+### Stifle / Trickbind / Torpor Orb / Dress Down
+
+These effects all seek to prevent the Thassa's Oracle trigger from either
+resolving or ever triggering in the first place. For any permanent based one you
+will need to remove it to be able to win from the Oracle trigger. For the
+instant speed ones, you will need some sort of interaction, either counter magic
+or discard (however in the case of {{Trickbind}} you have to be proactive thanks
+to the split second ability) so something like Duress is a must. You can also
```

Comment:
> something like Duress is a must

Some lists used to play Counterbalance in the sideboard so that could be mentioned 🤓

Reply:
Will probs leave out